react-app/src/components/form/TextFormField.tsx

```diff
@@ -37,7 +37,7 @@ const TextFormField = (props: TextFormFieldProps) => {
             value={value ?? defaultVal}
             fullWidth
             label={label}
-            type="text"
+            type={numeric ? 'number' : 'text'}
```

Comment:
Doing this may cause the numeric validation already implemented to not function as expected

Reply:
Reverted